### ISSUE TITLE
index collision problem

### ISSUE THREAD

**Issue by csachs** [cgohlke/roifile#2]
`ImagejRoi.frompoints` picks a random index to create a (I guess supposedly) unique name per ROI.
However it occurred multiple times for me in some batch processing jobs with many thousand ROIs, that collisions occurred.
I'm well aware that there is no easy solution to this.
Potential fixes would be to raise the bits of the random number to make a collision more unlinkely, or to have a singleton variable in the module as a ROI counter (still, e.g. ROIs arriving from multiple processes could have colliding IDs).
Do you think there is a good solution to this? I guess I will workaround this for now by providing an incremented `index` by my code. 

For example, 
```python
import numpy
numpy.random.seed(0)

def collide():
    set_ = set()
    while True:
        ri = numpy.random.randint(0, 2**31 - 1)
        if ri in set_:
            return len(set_)
        else:
            set_.add(ri)

for _ in range(5): print(collide())
``` 
Yields some not-so-super-high collission counts:
```
67205
64247
60800
105507
27671
```

**Comment by cgohlke** [comment]
The fix is to provide unique `name` or `index` arguments to the `frompoints` function.

The autonaming `index = numpy.random.randint(0, 2**31 - 1); name = f'F{self.t_position:02}-{index:x}'` was chosen to mimic names used by ImageJ. Since ImageJ ROIs are not documented, it's probably better to just use UUIDs here.

**Comment by cgohlke** [comment]
Should be fixed in v2020.11.28, which uses UUIDs for names.
